notebooks/11-xarray-intro.md

```diff
@@ -220,18 +103,14 @@ We could use the Numpy style of data slicing:
 herstappe[0]
```

Comment:
Just above, it still uses "data_file", but you renamed it to "herstappe_file"

Reply:
fixed